code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -74,17 +74,17 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	min_temp = 15
 	max_temp = 35
 	reaction_type = ENDOTHERMIC_REACTION
-	energy_release = 300000
-	requirements = list(/datum/gas/freon = 50, /datum/gas/plasma = 40, /datum/gas/oxygen = 30)
-	products = list(/obj/item/stack/sheet/hot_ice = 1)
+	energy_release = 3000000
+	requirements = list(/datum/gas/freon = 120, /datum/gas/plasma = 70, /datum/gas/oxygen = 30)
+	products = list(/obj/item/stack/sheet/hot_ice = 10)
```

Comment:
Doesn't this make more plasma reagent then it takes plasma?

Reply:
uh yeah i was thinking of only freon consumption, uhh is best if i lower the product amount and lower the freon a bit